boltkit/controller.py

```diff
@@ -713,7 +717,9 @@ def _execute_cluster_command(parsed):
     if command == "install":
         core_count = int(parsed.core_count)
         read_replica_count = int(parsed.read_replica_count)
-        path = cluster_ctrl.install(parsed.version, core_count, read_replica_count, parsed.password, parsed.verbose)
+        initial_port = int(parsed.initial_port)
```

Comment:
Perhaps we should handle `ValueError`s here (and in the pre-existing lines above).